common/djangoapps/edeos/api_calls.py

```diff
@@ -246,4 +252,8 @@ def referrals_store(self, payload):
         return self.call_api("referrals/store", payload)
 
     def statistics(self, payload):
-        return self.call_api("statistics", payload)
\ No newline at end of file
+        return self.call_api("statistics", payload)
+    
+    def profile_statistics(self, payload):
+        return self.call_api("profile_statistics", payload)
+        
```

Comment:
please fix the no newline at the end of file

Reply:
But it should be there? or shouldn't?

Reply:
there should. Check out the next file - there is no such error there